waku/v2/node/storage/message/sqlite_store/retention_policy_time.nim

```diff
@@ -0,0 +1,45 @@
+{.push raises: [Defect].}
```

Comment:
Same query re: other stores making use of these functions

Reply:
Check my comment: https://github.com/status-im/nwaku/pull/1141#issuecomment-1243721470